src/Configuration/Dotenv/Types.hs

```diff
@@ -30,4 +36,29 @@ defaultConfig =
     { configExamplePath = []
     , configOverride = False
     , configPath = [ ".env" ]
+    , configVerbose = False
     }
+
+
+newtype ReaderT r m a = ReaderT { runReaderT :: r -> m a }
+mapReaderT :: (m a -> n b) -> ReaderT r m a -> ReaderT r n b
+mapReaderT f m = ReaderT $ f . runReaderT m
+instance (Functor m) => Functor (ReaderT r m) where
+    fmap f  = mapReaderT (fmap f)
+instance (Applicative m) => Applicative (ReaderT r m) where
+    pure    = liftReaderT . pure
+    f <*> v = ReaderT $ \ r -> runReaderT f r <*> runReaderT v r
+
+instance (Monad m) => Monad (ReaderT r m) where
+    return   = liftReaderT . return
+    m >>= k  = ReaderT $ \ r -> do
+      a <- runReaderT m r
+      runReaderT (k a) r
+    m >> k = ReaderT $ \ r -> runReaderT m r >> runReaderT k r
+
+liftReaderT :: m a -> ReaderT r m a
+liftReaderT m = ReaderT (const m)
+{-# INLINE liftReaderT #-}
```

Comment:
@FranzGB Why `INLINE`?

Reply:
Woops I copied this function from a library and forgot to remove that